File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/ocso/OCSACT2GetPatientListOCSInfo.java
package nta.med.data.model.ihis.ocso;

public class OCSACT2GetPatientListOCSInfo {
	private String inOutGubun      ;
	private String bunho             ;
	private String suname            ;
	private String suname2           ;
	private String gwa               ;
	private String gwaName          ;
	private String doctor            ;
	private String doctorName       ;
	private String jundalTable      ;
	private String reserYn          ;
	private String emergency         ;
	private String trial             ;
	private Double pkocs1003         ;
	private Double pkout1001         ;
	public OCSACT2GetPatientListOCSInfo(String inOutGubun, String bunho, String suname, String suname2, String gwa,
			String gwaName, String doctor, String doctorName, String jundalTable, String reserYn, String emergency,
			String trial, Double pkocs1003, Double pkout1001) {
		super();
		this.inOutGubun = inOutGubun;
		this.bunho = bunho;
		this.suname = suname;
		this.suname2 = suname2;
		this.gwa = gwa;
		this.gwaName = gwaName;
		this.doctor = doctor;
		this.doctorName = doctorName;
		this.jundalTable = jundalTable;
		this.reserYn = reserYn;
		this.emergency = emergency;
		this.trial = trial;
		this.pkocs1003 = pkocs1003;
		this.pkout1001 = pkout1001;
	}
	public String getInOutGubun() {
		return inOutGubun;
	}
	public void setInOutGubun(String inOutGubun) {
		this.inOutGubun = inOutGubun;
	}
	public String getBunho() {
		return bunho;
	}
	public void setBunho(String bunho) {
		this.bunho = bunho;
	}
	public String getSuname() {
		return suname;
	}
	public void setSuname(String suname) {
		this.suname = suname;
	}
	public String getSuname2() {
		return suname2;
	}
	public void setSuname2(String suname2) {
		this.suname2 = suname2;
	}
	public String getGwa() {
		return gwa;
	}
	public void setGwa(String gwa) {
		this.gwa = gwa;
	}
	public String getGwaName() {
		return gwaName;
	}
	public void setGwaName(String gwaName) {
		this.gwaName = gwaName;
	}
	public String getDoctor() {
		return doctor;
	}
	public void setDoctor(String doctor) {
		this.doctor = doctor;
	}
	public String getDoctorName() {
		return doctorName;
	}
	public void setDoctorName(String doctorName) {
		this.doctorName = doctorName;
	}
	public String getJundalTable() {
		return jundalTable;
	}
	public void setJundalTable(String jundalTable) {
		this.jundalTable = jundalTable;
	}
	public String getReserYn() {
		return reserYn;
	}
	public void setReserYn(String reserYn) {
		this.reserYn = reserYn;
	}
	public String getEmergency() {
		return emergency;
	}
	public void setEmergency(String emergency) {
		this.emergency = emergency;
	}
	public String getTrial() {
		return trial;
	}
	public void setTrial(String trial) {
		this.trial = trial;
	}
	public Double getPkocs1003() {
		return pkocs1003;
	}
	public void setPkocs1003(Double pkocs1003) {
		this.pkocs1003 = pkocs1003;
	}
	public Double getPkout1001() {
		return pkout1001;
	}
	public void setPkout1001(Double pkout1001) {
		this.pkout1001 = pkout1001;
	}
	
}

File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Ocso/OCSACT2GetPatientListOCSInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Ocso
{
    public class OCSACT2GetPatientListOCSInfo
    {
        private String _inOutGubun;
        private String _bunho;
        private String _suname;
        private String _suname2;
        private String _gwa;
        private String _gwaName;
        private String _doctor;
        private String _doctorName;
        private String _jundalTable;
        private String _reserYn;
        private String _emergency;
        private String _trial;
        private String _pkocs1003;
        private String _pkout1001;

        public String InOutGubun
        {
            get { return this._inOutGubun; }
            set { this._inOutGubun = value; }
        }

        public String Bunho
        {
            get { return this._bunho; }
            set { this._bunho = value; }
        }

        public String Suname
        {
            get { return this._suname; }
            set { this._suname = value; }
        }

        public String Suname2
        {
            get { return this._suname2; }
            set { this._suname2 = value; }
        }

        public String Gwa
        {
            get { return this._gwa; }
            set { this._gwa = value; }
        }

        public String GwaName
        {
            get { return this._gwaName; }
            set { this._gwaName = value; }
        }

        public String Doctor
        {
            get { return this._doctor; }
            set { this._doctor = value; }
        }

        public String DoctorName
        {
            get { return this._doctorName; }
            set { this._doctorName = value; }
        }

        public String JundalTable
        {
            get { return this._jundalTable; }
            set { this._jundalTable = value; }
        }

        public String ReserYn
        {
            get { return this._reserYn; }
            set { this._reserYn = value; }
        }

        public String Emergency
        {
            get { return this._emergency; }
            set { this._emergency = value; }
        }

        public String Trial
        {
            get { return this._trial; }
            set { this._trial = value; }
        }

        public String Pkocs1003
        {
            get { return this._pkocs1003; }
            set { this._pkocs1003 = value; }
        }

        public String Pkout1001
        {
            get { return this._pkout1001; }
            set { this._pkout1001 = value; }
        }

        public OCSACT2GetPatientListOCSInfo() { }

        public OCSACT2GetPatientListOCSInfo(String inOutGubun, String bunho, String suname, String suname2, String gwa, String gwaName, String doctor, String doctorName, String jundalTable, String reserYn, String emergency, String trial, String pkocs1003, String pkout1001)
        {
            this._inOutGubun = inOutGubun;
            this._bunho = bunho;
            this._suname = suname;
            this._suname2 = suname2;
            this._gwa = gwa;
            this._gwaName = gwaName;
            this._doctor = doctor;
            this._doctorName = doctorName;
            this._jundalTable = jundalTable;
            this._reserYn = reserYn;
            this._emergency = emergency;
            this._trial = trial;
            this._pkocs1003 = pkocs1003;
            this._pkout1001 = pkout1001;
        }

    }
}